project/AppDependencies.scala

```diff
@@ -4,21 +4,30 @@ import sbt.Keys.libraryDependencies
 import sbt._
 
 object AppDependencies {
+  private val circeVersion = "0.12.0"
+
+  private val circeDependencies = Seq(
+    "io.circe" %% "circe-core",
+    "io.circe" %% "circe-generic",
+    "io.circe" %% "circe-parser",
+    "io.circe" %% "circe-yaml"
+  ).map(_ % circeVersion)
```

Comment:
Honestly, if it were me, I'd just put these in with the other compile dependencies as they're not re-used by the test dependencies atm, but it's not a hill I'll die on.